examples/breakout/platform/src/graphics/lowlevel/quad.rs

```diff
@@ -2,6 +2,7 @@
 
 /// A polygon with 4 corners
 #[derive(Copy, Clone)]
+#[repr(C)]
```

Comment:
Could you add a “safety” comment for the bytemuck impls of this struct as well, to note why repr C is needed?

Reply:
Yep! Good catch.